interfaces/Portalicious/src/app/components/page-layout/components/project-menu/project-menu.component.html

```diff
@@ -0,0 +1,6 @@
+<div
+  class="w-full bg-white px-6 pt-2 lg:px-24 [&_.p-menuitem-link-active]:!border-purple-700 [&_.p-menuitem-link]:border-transparent [&_.p-tabmenu-nav]:border-transparent [&_.p-tabmenuitem]:border-transparent"
```

Comment:
```suggestion
  class="w-full bg-white px-6 pt-2 lg:px-24 [&_.p-tabmenu-nav]:border-transparent [&_.p-tabmenu-nav_.p-tabmenuitem:not(.p-highlight)_.p-menuitem-link]:border-transparent"
```

This is the literal translation of the css rule as it was in styles.scss, with the addition of the `not` to avoid redifining the border-color on the selected item.

Indeed, this isn't pretty at all, but I think it's ok for now.